commands/jinja2_tags.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from importlib.metadata import PackageNotFoundError
+from importlib.metadata import version
+
+from jinja2_simple_tags import StandaloneTag
+
+
+class VersionTag(StandaloneTag):
+    tags = {"version_info"}
+
+    def _get_version(self):
+        try:
+            return version("dbt-copilot-tools")
+        except PackageNotFoundError:
```

Comment:
Is it possible to test this? Looks like we are testing the happy path via fixtures already, but we also handle this exception and return "UNKNOWN".

Reply:
It should be an edge case - it only happens if the code is run directly, instead of after installing the package.  I did consider manually parsing pyproject.toml to grab the version, but I don't see much value in it.  Your `--version` command also has the same issue.

Reply:
The reason why I mention it is because we have a similar method here that does parse `pyproject.toml`: https://github.com/uktrade/copilot-tools/blob/main/utils/check_pypi.py#L43

Reply:
Oh cool, I'll raise another PR and fix it in both places.

Reply:
OK, I went down a rabbit hole here.  I wasn't liking the way Poetry installs the package as a dependency so at one point I manually removed it, which then caused the `PackageNotFound` exception. 

I did some digging yesterday and found this: https://github.com/python-poetry/poetry/issues/800, which explains why poetry works this way.

But the upshot of it is, there's no scenario that `version()` should fail and require a fall back to parsing the toml file.  So I'm just going to remove the `try/except` block.

Reply:
Yeahhh, I was thinking we would ideally never run into that exception anyway. It might be worth bumping up the version as well, so this change gets published on to PyPI as well.